pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -36,7 +35,14 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props }) => {
+export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props } : {
+    dropdownItems: React.ReactNode,
+    position?: "start" | "end" | "right" | "left" | "center",
+    isDisabled?: boolean,
```

Comment:
This is also  dubious -- it's more likely that the caller explicitly passes a null-ish value, but that's usually out of laziness, not because it conceptually makes sense. If we do this at all, then let's make it a `bool`.

Reply:
This is just another case of "tsc doesn't understand default arguments" then. Also, my head keeps translating the `?` to "nullable"..